File: TypeScript/Class/Private.ts
function foo() {
    var x = 1;
    var y = 3;
    if (true) {
        var x = 2;
        var y_1 = 4;
        console.log(x);
        console.log(y_1);
    }
    console.log(x);
    console.log(y);
}

//https://pt.stackoverflow.com/q/323697/101

File: CSharp/Class/Private.cs
public class Classe {
    Temp temp = new Temp();
    public string Texto { get => temp.Str; set => temp.Str = value; }
    public int Valor { get => temp.I; set => temp.I = value; }

    private class Temp {
        public string Str { get; set; }
        public int I { get; set; }
    }
}

//https://pt.stackoverflow.com/q/290621/101
